src/rules/at-if-no-null/index.js

```diff
@@ -32,20 +32,20 @@ function rule(expectation) {
       }
 
       // If rule != null and (expr), skip
-      if (atrule.params.match(/.* != null and .*/)) {
+      if (/.* != null and .*/.test(atrule.params)) {
```

Comment:
I kept the `test` changes as a separate patch; let me know if you want me to drop them.